### ISSUE TITLE
Error parsing unicode characters in doujin titles when scraping chapters

### ISSUE THREAD

**Issue by azurenekowo** [4pii4/piehtvn#4]
For some doujins, ex. `/get-chapters?url=34544-doc-truyen-koibito-to-soto-de-h-shitara-shinsetsuna-yankī-ni-ntr-sa-reta-hanashi.html`

![image](https://github.com/4pii4/piehtvn/assets/67498765/d6539c90-04bb-4f52-b032-91a1254a715e)

**The issue**
The character `ī` (`U+012B`) is present in the doujin's `name`, but for some reason `/get-metadata` didn't return an error when `/get-chapters` did.

**Tracelog**
`UnicodeEncodeError('latin-1', 'https://hentaivn.red/list-showchapter.php?idchapshow=34544&idlinkanime=koibito-to-soto-de-h-shitara-shinsetsuna-yankī-ni-ntr-sa-reta-hanashi', 116, 117, 'ordinal not in range(256)')`

```
Traceback (most recent call last):
  File "C:\Users\Azure\AppData\Local\Programs\Python\Python310\lib\site-packages\bottle.py", line 876, in _handle
    return route.call(**args)
  File "C:\Users\Azure\AppData\Local\Programs\Python\Python310\lib\site-packages\bottle.py", line 1759, in wrapper
    rv = callback(*a, **ka)
  File "D:\piehtvn\backend.py", line 50, in backend_get_chapters
    return generate_response(doc.get_chapters())
  File "D:\piehtvn\piehtvn.py", line 212, in get_chapters
    response = requests.get(f'https://{self.domain}/list-showchapter.php', params=params, headers=headers)
  File "C:\Users\Azure\AppData\Local\Programs\Python\Python310\lib\site-packages\requests\api.py", line 73, in get
    return request("get", url, params=params, **kwargs)
  File "C:\Users\Azure\AppData\Local\Programs\Python\Python310\lib\site-packages\requests\api.py", line 59, in request
    return session.request(method=method, url=url, **kwargs)
  File "C:\Users\Azure\AppData\Local\Programs\Python\Python310\lib\site-packages\requests\sessions.py", line 589, in request
    resp = self.send(prep, **send_kwargs)
  File "C:\Users\Azure\AppData\Local\Programs\Python\Python310\lib\site-packages\requests\sessions.py", line 703, in send
    r = adapter.send(request, **kwargs)
  File "C:\Users\Azure\AppData\Local\Programs\Python\Python310\lib\site-packages\requests\adapters.py", line 486, in send
    resp = conn.urlopen(
  File "C:\Users\Azure\AppData\Local\Programs\Python\Python310\lib\site-packages\urllib3\connectionpool.py", line 790, in urlopen
    response = self._make_request(
  File "C:\Users\Azure\AppData\Local\Programs\Python\Python310\lib\site-packages\urllib3\connectionpool.py", line 496, in _make_request
    conn.request(
  File "C:\Users\Azure\AppData\Local\Programs\Python\Python310\lib\site-packages\urllib3\connection.py", line 394, in request
    self.putheader(header, value)
  File "C:\Users\Azure\AppData\Local\Programs\Python\Python310\lib\site-packages\urllib3\connection.py", line 308, in putheader
    super().putheader(header, *values)
  File "C:\Users\Azure\AppData\Local\Programs\Python\Python310\lib\http\client.py", line 1255, in putheader
    values[i] = one_value.encode('latin-1')
UnicodeEncodeError: 'latin-1' codec can't encode character '\u012b' in position 116: ordinal not in range(256)
```

**Comment by 4pii4** [comment]
Probably fixed as of [ed3227e](https://github.com/4pii4/piehtvn/commit/ed3227e65dbf20279778b2d2396ccea0198bbb40).

**Comment by azurenekowo** [comment]
Affirmative that this bug is fixed in the latest commit.
![image](https://github.com/4pii4/piehtvn/assets/67498765/797f1c23-50a1-404a-917a-8eacc7307749)

```http
GET /get-chapters?url=34544-doc-truyen-koibito-to-soto-de-h-shitara-shinsetsuna-yank%C4%AB-ni-ntr-sa-reta-hanashi HTTP/1.1
User-Agent: insomnium/0.2.3-a
Host: 127.0.0.1:7479
```

```http
HTTP/1.0 200 OK
Date: Tue, 09 Jan 2024 02:52:16 GMT
Server: WSGIServer/0.2 CPython/3.10.6
Access-Control-Allow-Origin: *
Content-Type: application/json
Content-Length: 180

[{"title": "Oneshot", "url": "34544-64074-xem-truyen-koibito-to-soto-de-h-shitara-shinsetsuna-yankī-ni-ntr-sa-reta-hanashi-oneshot", "time": 1693587600, "domain": "hentaivn.red"}]
```
